trunk/NDHMS/Routing/module_HYDRO_io.F

```diff
@@ -8948,6 +8978,20 @@ subroutine read_route_lake_netcdf(route_lake_file,                         &
     call get_1d_netcdf_real(ncid, 'OrificeE', ORIFICEE,  'read_route_lake_netcdf', .TRUE.)
     call get_1d_netcdf_real(ncid, 'lat', lakelat,        'read_route_lake_netcdf', .TRUE.)
     call get_1d_netcdf_real(ncid, 'lon', lakelon,        'read_route_lake_netcdf', .TRUE.)
+
+    ! read variables for lake_shape_option not equal to 0 (default Box shape)
+    if(nlst(did)%lake_shape_option /= 0) then
```

Comment:
This should probably be >= 0 so that pass-through doesn't require these variables to be defined.